atlasdb-container-test-utils/src/main/java/com/palantir/atlasdb/containers/Containers.java

```diff
@@ -132,6 +133,11 @@ private void setupDockerComposeRule() throws InterruptedException, IOException {
         dockerComposeRule.before();
     }
 
+    private void startCollectingLogs() throws IOException, InterruptedException {
```

Comment:
nit: These exceptions aren't being thrown by the body any more

Reply:
Removed.